tests/unit/__snapshots__/client.test.ts.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
whats this file? lets not include it? it looks like a complete different test mechanism than what we have in the SDK, might be good to revisit using it in the future.

Reply:
toMatchSnapshot is mostly good to use when we want to test against a UI and that it doesnt change unexpectedly, which is were we mostly use it (doc website, aptos-dashboard)

Reply:
I disagree. Snapshot based testing is useful for more than UI stuff. In fact we have various examples of snapshot based testing in our codebase that use it for backend code (see the search above), incl. with other libraries. We also use snapshot based testing in our rust codebase in aptos-core (we call this golden files there but it's the same concept).
Furthermore you can argue that the error messages are sort of the UI surface of the SDK since that's what users/devs will see as output when working with it.